web/client/plugins/SecurityPopup.jsx

```diff
@@ -65,12 +65,19 @@ function SecurityPopup({
 
     const id = services[currentFormIndex]?.protectedId;
     const show = currentFormIndex + 1 < services.length;
-    const currentCreds = getCredentials(id);
-    const [creds, setCreds] = useState(currentCreds);
+    const [creds, setCreds] = useState({});
 
     useEffect(() => {
-        setCreds(currentCreds);
-    }, [id]);
+        const credentials = getCredentials(id);
+        setCreds(credentials);
+    }, [id, currentFormIndex]);
+    useEffect(() => {
+        const credentials = getCredentials(id);
+        setCreds(credentials);
+        return () => {
+            setCreds({});
+        };
+    }, []);
```

Comment:
I tested locally and I found this issue, where the form is empty after saving:

https://github.com/user-attachments/assets/5994678e-4607-4e8b-a6ac-c0fa29116dee

Here I opened a PR with a proposal to change the management of credential directly inside the popup to take advantage of mount unmount event triggered by the key prop https://github.com/MV88/MapStore2/pull/7

Please double check if the changes included in the PR works as expected

Reply:
> I tested locally and I found this issue, where the form is empty after saving:
> 
>  security-empty.mp4 
> Here I opened a PR with a proposal to change the management of credential directly inside the popup to take advantage of mount unmount event triggered by the key prop [MV88#7](https://github.com/MV88/MapStore2/pull/7)
> 
> Please double check if the changes included in the PR works as expected

i fixed also tests. you can review again